worlds/civ_6/__init__.py

```diff
@@ -0,0 +1,205 @@
+            # Add one less than the total number of eras (start in ancient, don't need to find it)
+            for era in EraType:
+                if era.value == "ERA_ANCIENT":
+                    continue
+                progressive_era_item = self.item_table.get("Progressive Era")
+                if progressive_era_item:
+                    self.multiworld.itempool += [self.create_item(progressive_era_item.name)]
+                else:
+                    logging.error("Progressive Era item not found in item_table.")
+            self.multiworld.early_items[self.player]["Progressive Era"] = 2
+
+        num_filler_items = 0
+        # Goody items, create 10 by default if options are enabled
+        if self.options.shuffle_goody_hut_rewards:
+            num_filler_items += 10
+
+        if self.options.boostsanity:
+            num_filler_items += len(get_boosts_data())
+
+        filler_count = {rarity: math.ceil(FILLER_DISTRIBUTION[rarity] * num_filler_items) for rarity in FillerItemRarity.__reversed__()}
+        min_count = 1
+        # Add filler items by rarity
+        total_created = 0
+        for rarity, count in filler_count.items():
+            for _ in range(max(min_count, count)):
+                if total_created >= num_filler_items:
+                    break
+                self.multiworld.itempool += [self.create_item(
+                    get_random_filler_by_rarity(self, rarity).name)]
+                total_created += 1
+
+    def post_fill(self) -> None:
+        if self.options.pre_hint_items == "none":
+            return
+
+        show_flags = {
+            ItemClassification.progression: self.options.pre_hint_items != "none",
+            ItemClassification.useful: self.options.pre_hint_items == "no_junk" or self.options.pre_hint_items == "all",
+            ItemClassification.filler: self.options.pre_hint_items == "all",
+        }
+
+        start_location_hints: Set[str] = self.options.start_location_hints.value
+        for location_name, location_data in self.location_table.items():
+            if location_data.location_type != CivVICheckType.CIVIC and location_data.location_type != CivVICheckType.TECH:
+                continue
+
+            location: CivVILocation = self.get_location(location_name)  # type: ignore
+
+            if not location.item or not show_flags.get(location.item.classification, False):
+                continue
+
+            start_location_hints.add(location_name)
+
+    def fill_slot_data(self) -> Dict[str, Any]:
+        return {
+            "progression_style": self.options.progression_style.value,
+            "death_link": self.options.death_link.value,
+            "research_cost_multiplier": self.options.research_cost_multiplier.value,
+            "death_link_effect": self.options.death_link_effect.value,
+            "death_link_effect_percent": self.options.death_link_effect_percent.value,
+
+        }

```

Comment:
Could do this instead (although you'll want to double check the values are the same format):
```suggestion
        return self.options.as_dict(
            "progression_style",
            "death_link",
            "research_cost_multiplier",
            "death_link_effect",
            "death_link_effect_percent",
        )
```

Reply:
Nice, confirmed they translate as expected